src/transformers/models/gpt_neo/modeling_gpt_neo.py

```diff
@@ -55,6 +55,8 @@
 # This makes `_prepare_4d_causal_attention_mask` a leaf function in the FX graph.
 # It means that the function will not be traced through and simply appear as a node in the graph.
 if is_torch_fx_available():
+    import torch.fx
```

Comment:
we can either to have this under `if not is_torch_greater_or_equal_than_1_12`, or just a comment to say this is necessary for torch <= 1.12

(no big deal, but at least it ease our life when we drop the support of torch 1.12 in the future)

Reply:
Good idea! Tbh, I didn't know all of the pytorch versions this failed for. I just checked and the import passed with 1.13. I'll add a conditional import :)

Reply:
probably `is_torch_greater_or_equal_than_1_12` is not correct one : I should say `is_torch_greater_or_equal_than_1_13` (but we have to create this entry first)

sorry for this

Reply:
Done!